### PR TITLE
Drop chai and mocha deps

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,7 @@
     "lint": "eslint . --ext .ts,.tsx",
     "format": "prettier --write **/*.ts",
     "docs": "rm -rf docs/ && typedoc --plugin typedoc-plugin-markdown",
-    "test": "mocha -r ts-node/register -t 120s 'tests/**/*.test.ts'",
-    "all": "npm run format && npm run lint && npm run docs && npm run build"
+    "test": "bash ./bin/runTests.sh"
```
Comment (verbanicm) [inline package.json:12]: Why? Cant we filter on a suffix here?

Comment (verbanicm) [reply package.json:12]: @sethvargo

**Review by verbanicm — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (sethvargo) [reply package.json:12]: What do you mean by "filter on a suffix"?

Comment (verbanicm) [reply package.json:12]: In auth, we inline the command, why do we need a separate bash script here?

e.g. https://github.com/google-github-actions/auth/blob/main/package.json#L10

**Review by verbanicm — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (verbanicm) [reply package.json:12]: Thanks!

Comment (sethvargo) [reply package.json:12]: I added a really long comment, but the short answer is "Windows". I should have done this in auth too (I'll backport eventually).

**Review by verbanicm — APPROVED** [review]

**Pull request merged**